### ISSUE TITLE
Equalizer for reverb wet signal

### ISSUE THREAD

**Issue by AS383** [calf-studio-gear/calf#126]
I often use a Calf Eq 5 after reberb effect. It will be nice to have some parametric equalization for reverb wet signal. This feature can help to give more presence for vocal with reverb for example.

**Comment by unfa** [comment]
There is already a LP/HP filter pair in Calf Reverb - you'd need an additional single-band  (semi-)parametric EQ?

**Comment by AS383** [comment]
Yes. I need a parametric EQ with one or two bands. If it have some on/off button it would be great (can help with A/B test of sound).
